### ISSUE TITLE
May I ask question??

### ISSUE THREAD

**Issue by roger865477** [andabi/deep-voice-conversion#8]
Could you tell me that what percentage is your Net1's accuracy ?
Because I run your 'train1.py' code with the TIMIT dataset , I found that the accuracy is only about 65%.

Thank you.

**Comment by andabi** [comment]
@roger865477 I also got 65~75% accuracy in train1. I found train2 worked quite well even when train1 accuracy of around 70%. But I'm still researching to improve the train1 accuracy. I welcome your idea. Thanks.

**Comment by emmacirl** [comment]
@andabi
 Hi，I run 'train1.py' with TIMIT and I set  learn rate lr=0.0003, num_epochs=1000 (I use the default settings). I found that the accauracy is only 0.44 and the loss is 1.8. What is the configuration of your net1 ?
